Classes/Domain/Model/LanguageSkill.php

```diff
@@ -10,6 +10,7 @@
 
 /**
  * LanguageSkill model - connects languages and their knowledge level to applications
+ * @validate \PAGEmachine\Ats\Domain\Validator\LanguageSkillValidator
```

Comment:
You don't need this line [according to the docs](https://docs.typo3.org/typo3cms/ExtbaseFluidBook/9-CrosscuttingConcerns/2-validating-domain-objects.html#validating-in-the-domain-model-with-an-own-validator-class) since the name convention is enough.